test/e2e/network_test.go

```diff
@@ -297,6 +298,23 @@ var _ = Describe("Podman network", func() {
 		Expect(rmAll).Should(ExitCleanly())
 	})
 
+	It("podman run container host interface name", func() {
+		Skip("We need to update netavark first.")
+
+		ctrName := "testCtr"
+		vethName := "my_veth"
+		container := podmanTest.Podman([]string{"run", "-dt", "--network", "bridge:host_interface_name=" + vethName, "--name", ctrName, ALPINE, "top"})
+		container.WaitWithDefaultTimeout()
+		Expect(container).Should(ExitCleanly())
+
+		if !isRootless() {
+			// make sure cni/netavark created bridge with expected name
```

Comment:
oh and please drop cni here, we no longer support cni upstream and cni doesn't support this option